cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -150,14 +200,16 @@ label:deprecated[]
 SHOW BTREE INDEXES
 ----
 a|
-B-tree indexes are deprecated and need to be migrated before upgrading to 5.0.
+B-tree indexes are deprecated, partially replaced for now and will be fully replaced in 5.0 by <<indexes-future-indexes, Future indexes>>.
```

Comment:
```suggestion
B-tree indexes are deprecated, partially replaced for now, and will be fully replaced in 5.0 by <<indexes-future-indexes, future indexes>>.
```
This addition did the trick, but I'm not sure it should be capitalized now when I see it. Also, I think there should be a comma. Sorry about that.